draco/core/containers.py

```diff
@@ -438,18 +446,80 @@ def axes(self):
         axes = set(self._class_axes())
 
         # Add in any axes found on the instance (this is needed to support the table
-        # classes where
-        # the axes get added at run time)
+        # classes where the axes get added at run time)
         axes |= set(self.__dict__.get("_axes", []))
 
         # This must be the same order on all ranks, so we need to explicitly sort to
         # get around the hash randomization
         return tuple(sorted(axes))
 
+    def _sanitize_axis_map(self, axis, axis_map, axes_from):
```

Comment:
This is a pretty iffy way to do this check. I think that I've removed any cases where an axis is set in kwargs directly from a property (i.e. time=data.time[:]), so we can probably remove this, but I wonder about having some way to warn a user about this specific case.